### PR TITLE
JavaScript -> TypeScript

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

File: js/src/renderer.ts [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import * as widgets  from '@jupyter-widgets/base';
-import Handsontable from 'handsontable';
+import * as Handsontable from 'handsontable/dist/handsontable.full.js';
```
Comment (martinRenou) [inline js/src/renderer.ts]: Note to self: This line is not fine

File: js/src/labplugin.ts [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
-var base = require('@jupyter-widgets/base');
-var version = require('../package.json').version;
+import * as sheet from './sheet';
+import * as renderer from './renderer';
+import * as base from '@jupyter-widgets/base';
+import * as pkg from '../package.json';
 
-module.exports = [
+
+export [
```
Comment (martinRenou) [inline js/src/labplugin.ts]: I need help on this one, how to export a list?

File: js/src/extension-renderer.ts [hunk 3]
```diff
@@ -14,6 +14,4 @@ if (window.require) {
 }
```
Comment (martinRenou) [inline js/src/extension-renderer.ts]: The TypeScript transpiler complains on this file that `window.require` does not exist.

**Comment by martinRenou** [comment]
@maartenbreddels :rocket:

**Comment by maartenbreddels** [comment]
Great! 
Oone thing I thought about, Maybe we should include the jupyterlab build as well in travis. Since the build takes a long time, maybe we should only do that on linux, what do you think?

**Comment by martinRenou** [comment]
For some reason, installing the jupyterlab extension build takes a lot of time with Python2.7, I'll just enable the jupyterlab extension build for linux and Python != 2.7

**Comment by martinRenou** [comment]
@maartenbreddels Done :)

**Comment by maartenbreddels** [comment]
LGTM, I'll let you merge it! :)

**Pull request merged**